src/lib/accessibility/Behaviors/Toolbar/ToolbarBehavior.ts

```diff
@@ -1,11 +1,17 @@
-import { Accessibility } from '../../interfaces'
+import { Accessibility, FocusZoneMode } from '../../interfaces'
 
 const ToolbarBehavior: Accessibility = {
   attributes: {
     root: {
       role: 'toolbar',
     },
   },
+  focusZone: {
+    mode: FocusZoneMode.Wrap,
+    props: {
+      isCircularNavigation: false,
+    },
```

Comment:
What about setting direction? As far as I remember it will be bidirectional by default (which has bigger performance impact). Might be worth to set to the horizontal direction.

Reply:
this is just initial implementation, we will need to get feedback on the arrow keys. For user that can see, that a toolbar is vertical/horizontal it makes sense to use only up/down or left/right. but screen reader user does not see the orientation of the toolbar/tabbar so it might be confusing for him.